draft-ietf-dhc-addr-notification.md

```diff
@@ -230,8 +231,8 @@ Unlike other types of messages, which are sent from the link-local address of th
 
 On clients with multiple interfaces, the client MUST only send the packet on the network interface that has the address being registered, even if it has multiple interfaces with different addresses. If the same address is configured on multiple interfaces, then the client MUST send ADDR-REG-INFORM each time the address is configured on an interface that did not previously have it, and refresh each registration independently from the others.
 
-The client MUST only send the ADDR-REG-INFORM message for valid ({{!RFC4862}}) addresses of global scope ({{!RFC4007}}). This includes ULA addresses, which are defined in {{!RFC4193}} to have global scope.
-The client MUST NOT send the  ADDR-REG-INFORM message for addresses configured by DHCPv6.
+The client MUST only send the ADDR-REG-INFORM message for valid ({{!RFC4862}}) addresses of global scope ({{!RFC4007}}). This includes ULA addresses, which are defined in {{!RFC4193}} to have global scope. This also includes statically assigned addresses of global scope (such addresses are considered to be valid indefinitely).
```

Comment:
I'm not sure it's a good idea to say addresses last forever....

Reply:
Let's sort it out tomorrow - I'm merging meantime.